packages/react-native-web/src/modules/ImageLoader/index.js

```diff
@@ -118,30 +129,70 @@ const ImageLoader = {
   has(uri: string): boolean {
     return ImageUriCache.has(uri);
   },
-  load(uri: string, onLoad: Function, onError: Function): number {
+  load(source: ImageSource, onLoad: Function, onError: Function): number {
     id += 1;
     const image = new window.Image();
+
     image.onerror = onError;
-    image.onload = (e) => {
+    image.onload = () => {
       // avoid blocking the main thread
-      const onDecode = () => onLoad({ nativeEvent: e });
```

Comment:
> Looks like this would break any web platform code where `onLoad` events are expecting the `nativeEvent`?

This was moved to the Image component
We still call `onLoad` with an object containing `nativeEvent` here:

https://github.com/Expensify/react-native-web/blob/9edf19bb7567b4a61f043c716f7384ebdeb0726b/packages/react-native-web/src/exports/Image/index.js#L345

though the structure did change (noted in other comment)

Reply:
I've updated the code and avoided this breaking change
I guess I just missed to see that `source` was being successfully added on the `event` object 

Now it would be up to the maintainer to decide whether they want to leave just `source` or keep the rest of the data